packages/terra-demographics-banner/src/DemographicsBanner.module.scss

```diff
@@ -106,5 +109,10 @@
     .identifiers {
       text-align: right;
     }
+
+    .row {
+      flex: 100%;
```

Comment:
This does not appear to be a valid value based on the documentation for this property.

Reply:
This is a shorthand for the `flex-basis` property supported by modern browsers, but apparently not IE 10. I found an alternative that should work for IE and will include that in my next commit.